src/assets/i18n/de.json

```diff
@@ -1273,7 +1273,12 @@
     "tag_id_already_used": "Der Badge '{{tagID}}' existiert bereits",
     "redirect": "Zu den Badges navigieren",
     "title": "Badge",
-    "default_tag": "Default badge"
+    "default_tag": "Default badge",
+    "delete_tags_title": "Delete Badge(s)",
+    "delete_tags_confirm": "Do you really want to delete '{{quantity}}' badge(s)?",
+    "delete_tags_success": "'{{inSuccess}}' badges have been deleted successfully",
+    "delete_tags_partial": "'{{inSuccess}}' badges  have been deleted successfully and '{{inError}}' badges have encountered an error, check the logs",
```

Comment:
Extra space to remove.

Reply:
@mahdibenromdhane: if you install the recommended plugins in vscode and ensure they are all enabled, you not have that kind of formatting issue.

Reply:
> @mahdibenromdhane: if you install the recommended plugins in vscode and ensure they are all enabled, you will not have that kind of formatting issue.

I thought I have them, I will double check. Thanks.